test-lit/semantics/separate-module-procs.f90

```diff
@@ -7,6 +7,9 @@
 !===------------------------------------------------------------------------===
```

Comment:
What is this line? It comes from the old file, but any way very odd, no?

Reply:
It is very odd...